promises/promise.go

```diff
@@ -9,23 +9,33 @@ import (
 	"github.com/mysteriumnetwork/payments/identity"
 )
 
-type Promise struct {
-	ServiceConsumer common.Address
-	Receiver        common.Address
-	SeqNo           int64
-	Amount          int64
+type ExtraData interface {
+	Hash() []byte
 }
 
-const issuerPrefix = "Issuer prefix:"
+const emptyExtra = "emptyextra"
+
+type EmptyExtra struct {
```

Comment:
it's not really empty though, is it? maybe call it default?

Reply:
Yes it is :) because no variables to hash :) just a static string